templates/strapi/files/extensions/users-permissions/config/jwt.json

```diff
@@ -0,0 +1,3 @@
+{
+  "jwtSecret": "1dcd9d8e-9e0b-4f84-bada-6683b1cf177d"
```

Comment:
This seems like something we shouldn't be committing.  Can it be built off of the PLATFORM_ENTROPY env var?